oom-frontend/src/app/shared/modules/collaborator/collaborator-editor/collaborator-editor.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { CollaboratorEditorComponent } from './collaborator-editor.component';
+
+describe('CollaboratorEditorComponent', () => {
+  let component: CollaboratorEditorComponent;
+  let fixture: ComponentFixture<CollaboratorEditorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ CollaboratorEditorComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CollaboratorEditorComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Out-Of-Memory-Team/out-of-memory/pullRequest?prid=7533382)
```suggestion
  it("should create", () => {
```